fancy/analysis/analysis.py

```diff
@@ -178,30 +193,92 @@ def build_tables(
         if fit_only:
 
             # logarithmically spcaed array with 60% of points between KAPPA_MIN and 100
-            kappa_first = np.logspace(
-                np.log(1), np.log(10), int(num_points * 0.7), base=np.e
-            )
-            kappa_second = np.logspace(
-                np.log(10), np.log(100), int(num_points * 0.2) + 1, base=np.e
-            )
-            kappa_third = np.logspace(
-                np.log(100), np.log(1000), int(num_points * 0.1) + 1, base=np.e
-            )
-            kappa = np.concatenate(
-                (kappa_first, kappa_second[1:], kappa_third[1:]), axis=0
-            )
+            # kappa_first = np.logspace(
+            #     np.log(1), np.log(10), int(num_points * 0.7), base=np.e
+            # )
+            # kappa_second = np.logspace(
+            #     np.log(10), np.log(100), int(num_points * 0.2) + 1, base=np.e
+            # )
+            # kappa_third = np.logspace(
+            #     np.log(100), np.log(1000), int(num_points * 0.1) + 1, base=np.e
+            # )
+            # kappa = np.concatenate(
+            #     (kappa_first, kappa_second[1:], kappa_third[1:]), axis=0
+            # )
+            kappa = np.logspace(0, 5, num_points)
 
             # full table for fit
-            if parallel:
-                self.tables.build_for_fit_parallel(kappa)
+            if self.analysis_type == self.gmf_type:
+
+                kappa_gmf = []
+                alpha_approx = 2.5
+
+                # in the future, read Rex from the files
+                Eex = 2 ** (1 / (alpha_approx - 1)) * self.model.Eth
```

Comment:
Does this make sense with e.g. the source composition spectrum that we have implemented?

Reply:
No it doesnt, but this is contained in the analysis type for the GMF model (our previous case without the source composition spectrum). With the analysis type including the new composition model (`join_composition`), it will not pass through here and rather use the Rex obtained from tabulated values

Reply:
Thanks, got it